modules/apps/oauth2-provider/oauth2-provider-client/.eslintrc.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+module.exports = {
+	globals: {
+		CryptoJS: true,
+		bootstrap: true,
```

Comment:
Keys should be sorted

Reply:
ESLint: Expected object keys to be in natural ascending order. 'CryptoJS' should be before 'bootstrap'.(sort-keys)

Reply:
Didn't know that, thanks for the info.